GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
```

Comment:
> In other words, this seems a bit too vague.

In my opinion, that's a feature, not a bug. Attempting to codify too much specificity in advance invites low-value bike-shedding now and gaming of the system later. It provides little benefit beyond specificity-for-specificity's sake. The statement as it stands may be vague in terms of metrics, but I think it's also clear in terms of expectations: CTC members are expected to participate.

> It also seems to be in contradiction with this line above:

`these rules` in that line would include this text. Regardless, that line is about prerequisites. This new material is about expectations of standing members. I don't see a contradiction. (Perhaps there is room for improving phrasing to make things a bit more clear.)